database/PostgreSQL/seed.js

```diff
@@ -0,0 +1,23 @@
+const { Pool } = require('pg');
+const PASSWORD = require('../passwordIgnore.js');
+const path = require('path');
+
+const pool = new Pool({
+  user: 'postgres',
+  host: 'localhost',
+  database: 'images',
+  password: PASSWORD.PASSWORD,
+  port: 5432
+});
```

Comment:
Same thing here... rather than recreating all this could you import the existing pool from your database.js file?

Reply:
True. I might fix this in a new version.